packages/inter-protocol/src/interchainPool.js

```diff
@@ -1,26 +1,18 @@
 import { E, Far } from '@endo/far';
```

Comment:
I suppose we should delete this whole contract.

Its only purpose is to support permissionless creation of AMM pools.